build-corpus/grab-repos.js

```diff
@@ -0,0 +1,48 @@
+import fetch from 'node-fetch';
+import fs from 'fs';
+
+import { processArgs } from '../lib/process-args.js';
+import { join } from 'path';
+
+const args = processArgs(process.argv);
+
+const URL = `https://api.github.com/search/repositories?q=language:${
+  args.language
+}&stars:%3E0&sort=stars&per_page=${args.count || 25}`;
```

Comment:
I'm not actually sure template strings support multi-line interpolation like this.

Reply:
they do, this is just formatting by prettier.  everything between `{` and `}` have the same whitespace rules as code outside of a template.

and it works

Reply:
oh! TIL